LSP-rome.sublime-settings

```diff
@@ -0,0 +1,11 @@
+{
+    "selector": "source.json | source.js | source.ts | source.jsx | source.tsx | source.js.jsx | source.js.react | source.ts.react",
```

Comment:
Enabled json which will only be supported from the next rome version (supposedly). We need to wait for the next verson anyway as current has broken handling of change notifications.